tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "lib": ["es2017"],
+    "typeRoots": ["node_modules/@types"],
+    "rootDir": "./lib",
+    "outDir": "./build",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
```

Comment:
Fixes the imports of ES modules that have the "default" export. Doesn't require to write:

```js
const foo = require('module').default
```